src/extensions/sonarcloud/tasks/SonarCloudAnalyze/v1/task.json

```diff
@@ -16,25 +16,23 @@
   "deprecationMessage": "This task is deprecated. Please upgrade to the latest version. For more information, refer to https://docs.sonarsource.com/sonarcloud/advanced-setup/ci-based-analysis/sonarcloud-extension-for-azure-devops/",
   "minimumAgentVersion": "2.144.0",
   "demands": ["java"],
-  "inputs": [
-    {
-      "name": "jdkversion",
-      "type": "pickList",
-      "label": "JDK version source for analysis",
-      "defaultValue": "JAVA_HOME_17_X64",
-      "required": true,
-      "options": {
-        "JAVA_HOME": "Use JAVA_HOME",
-        "JAVA_HOME_11_X64": "Use built-in JAVA_HOME_11_X64 (hosted agent)",
-        "JAVA_HOME_17_X64": "Use built-in JAVA_HOME_17_X64 (hosted agent)",
-        "JAVA_HOME_21_X64": "Use built-in JAVA_HOME_21_X64 (hosted agent)"
-      },
-      "helpMarkDown": "Select the wanted Java version for the analysis : You can choose with either Self provided JAVA_HOME which will pick up the value of this env variable, or you can choose the built-in JAVA_HOME_XX_X64 value on hosted agent. \nDefault value is JAVA_HOME_17_X64, however if you choose either of the proposed value and they are not available, JAVA_HOME value will be picked up instead."
-    }
-  ],
+  "inputs": [{
```

Comment:
what caused the format change?

Reply:
same issue than the patch version, fixed it ty 😇